src/webgpu/api/validation/queue/copyImageBitmapToTexture.spec.ts

```diff
@@ -0,0 +1,409 @@
+export const description = `
+copyImageBitmapToTexture Validation Tests in Queue.
+
+Test Coverage:
+- For source.imagBitmap:
+  - imageBitmap generated from ImageData:
+    - Check that an error is generated when imageBitmap is closed. Otherwise, no error should be
+      generated.
+
+- For destination.texture:
+  - For 2d destination textures:
+    - Check that an error is generated when texture is in destroyed state. Otherwise, no error should
+      be generated.
+    - Check that an error is generated when texture is created without usage COPY_DST. Otherwise,
+      no error should be generated.
+    - Check that an error is generated when sample count is not 1. Otherwise, no error should be
+      generated.
+    - Check that an error is generated when mipLevel is too large. Otherwise, no error should be
+      generated.
+    - Check that an error is generated when texture format is not valid. Otherwise, no error should
+      be generated.
+
+- For copySize:
+  - Check that an error is generated when copySize has 0 in width, height or depth. Otherwise,
+    no error should be generated.
+  - Check that an error is generated when destination.texture.origin + copySize is too large.
+    Otherwise, no error should be generated.
+
+TODO: more test to cover source.imageBitmap generated from different source.
```

Comment:
Shouldn't that be in execution tests? Or maybe a single test that checks it is valid for all kind of sources.